README.md

```diff
@@ -8,3 +9,5 @@ Homeworks:
 
 
 Please refer to wiki for details.
+=======
```

Comment:
This file should NOT be modified.

Reply:
I didn't

Reply:
This is in YOUR pull request, so it somehow happened, probably by mistake.
Now you need to remove this extra change.
In order to do it - you have to inspect all your commits and find the problematic one.
Then do interactive rebase with stop on the problematic commit and fix it.